DOCS-1070: Pt5: Update UI on more tutorials (#1868)
viamrobotics/docs: docs/tutorials/get-started/servo-mousemover.md

First, install `viam-server` according to our [installation guide](/installation/)

Next, run this command in your Raspberry Pi terminal to install the pip package manager.

> Select "yes" when it asks if you want to continue.

Reviewer: ```suggestion
Select "yes" when asked if you want to continue.
```

Go to the [Viam app](https://app.viam.com), and navigate to the **Config** tab.

### Board component

Create a [board component](/components/board/):

Navigate to the **Components** subtab of your robot's **Config** tab.

Click **Create component** in the lower-left corner.

Select `board` for type and `pi` for model.

Enter `local` as the name for your board, then click **Create**.

You can name the board whatever you want.
You will reference it later.

### Servo component

Create a [servo component](/components/servo/):

Click **Create component** in the lower-left corner.

Select `servo` for type and `pi` for model.

Enter `fsr90r` as the name for your servo, then click **Create**.

> 

Author: ```suggestion
```

Go to the [Viam app](https://app.viam.com), and navigate to the **Config** tab.

### Board component

Create a [board component](/components/board/):

Navigate to the **Components** subtab of your robot's **Config** tab.

Click **Create component** in the lower-left corner.

Select `board` for type and `pi` for model.

Enter `local` as the name for your board, then click **Create**.

You can name the board whatever you want.

> You will reference it later.

Reviewer: What do you think of deleting these lines outright? We are already proposing a value for the board name. For tutorial context, I think it is appropriate to use an example name throughout: `local`.

Same with `FS90R` below.

Author: I think that, since the board name is used in configuring the servo but not in the code, that it's fine to keep this note in as a learning moment?

Reviewer: Alright that works. I'm just trying to avoid this kind of language:
"this value you entered does not matter"
"but it's important at an unspecified later date, don't forget"

Goal: names are up to you! But for this specific tutorial, use this one in particular
Actual: an aside about naming not mattering in the middle of actionable steps.

Feel free to retain if you find it helpful, I find it potentially convoluting!

### Servo component

Create a [servo component](/components/servo/):

Click **Create component** in the lower-left corner.

Select `servo` for type and `pi` for model.

> Enter `FS90R` as the name for your servo, then click **Create**.

Author: ```suggestion
Enter `fsr90r` as the name for your servo, then click **Create**.
```

We named the servo using the model number, but you can use whatever you want.

> You will reference it later.

Reviewer: Same comment as above: recommend skipping this.

Author: ```suggestion
```
Okay. If it weren't used in the code I'd say leave this in so users learn about naming, but it is in the code so I'll take it out to avoid confusion.

Author: Also, this tutorial uses two different names for the servo in different places, and then the code repo has a 3rd name, so I will make them consistent now :)

Reviewer: Thank you!!
